api/src/test/java/org/openmrs/module/fhir2/api/dao/impl/FhirFhirTaskDaoImplTest.java

```diff
@@ -23,13 +23,13 @@
 import org.hl7.fhir.r4.model.ServiceRequest;
 import org.junit.Before;
 import org.junit.Test;
-import org.openmrs.module.fhir2.Task;
+import org.openmrs.module.fhir2.FhirTask;
 import org.openmrs.module.fhir2.TestFhirSpringConfiguration;
 import org.openmrs.test.BaseModuleContextSensitiveTest;
 import org.springframework.test.context.ContextConfiguration;
 
 @ContextConfiguration(classes = TestFhirSpringConfiguration.class, inheritLocations = false)
-public class FhirTaskDaoImplTest extends BaseModuleContextSensitiveTest {
+public class FhirFhirTaskDaoImplTest extends BaseModuleContextSensitiveTest {
 	
```

Comment:
Remove prefix Fhir

Reply:
@corneliouzbett Oh really sorry I must have seen this before sending a PR. I'll fix this